src/rpc/methods/eth/types.rs

```diff
@@ -309,6 +311,44 @@ impl TryFrom<EthCallMessage> for Message {
     }
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema, PartialEq, Eq, Hash)]
+pub struct EthHash(#[schemars(with = "String")] pub H256);
```

Comment:
There is already some existing `eth::Hash` definition in our codebase. Let's rename it to `EthHash` and move it to `types.rs` to have a single definition.

Reply:
corrected